--- litex_setup.py	2025-03-15 14:55:21
+++ _litex_setup.py	2025-03-15 14:55:11
@@ -283,8 +283,7 @@ def litex_setup_install_repos(config="standard", user_
         if repo.develop:
             print_status(f"Installing {name} Git repository...")
             os.chdir(os.path.join(current_path, name))
-            subprocess.check_call("\"{python3}\" -m pip install --editable . {options}".format(
-                python3 = sys.executable,
+            subprocess.check_call("uv pip install --editable . {options}".format(
                 options = "--user" if user_mode else "",
                 ), shell=True)
     if user_mode:
