diff --git a/Makefile b/Makefile
index 8d5a90344..6832f56b0 100644
--- a/Makefile
+++ b/Makefile
@@ -110,30 +110,6 @@ AWK ?= awk
 
 ifeq ($(OS), Darwin)
 PLUGIN_LDFLAGS += -undefined dynamic_lookup
-
-# homebrew search paths
-ifneq ($(shell :; command -v brew),)
-BREW_PREFIX := $(shell brew --prefix)/opt
-$(info $$BREW_PREFIX is [${BREW_PREFIX}])
-ifeq ($(ENABLE_PYOSYS),1)
-CXXFLAGS += -I$(BREW_PREFIX)/boost/include/boost
-LDFLAGS += -L$(BREW_PREFIX)/boost/lib
-endif
-CXXFLAGS += -I$(BREW_PREFIX)/readline/include
-LDFLAGS += -L$(BREW_PREFIX)/readline/lib
-PKG_CONFIG_PATH := $(BREW_PREFIX)/libffi/lib/pkgconfig:$(PKG_CONFIG_PATH)
-PKG_CONFIG_PATH := $(BREW_PREFIX)/tcl-tk/lib/pkgconfig:$(PKG_CONFIG_PATH)
-export PATH := $(BREW_PREFIX)/bison/bin:$(BREW_PREFIX)/gettext/bin:$(BREW_PREFIX)/flex/bin:$(PATH)
-
-# macports search paths
-else ifneq ($(shell :; command -v port),)
-PORT_PREFIX := $(patsubst %/bin/port,%,$(shell :; command -v port))
-CXXFLAGS += -I$(PORT_PREFIX)/include
-LDFLAGS += -L$(PORT_PREFIX)/lib
-PKG_CONFIG_PATH := $(PORT_PREFIX)/lib/pkgconfig:$(PKG_CONFIG_PATH)
-export PATH := $(PORT_PREFIX)/bin:$(PATH)
-endif
-
 else
 LDFLAGS += -rdynamic
 ifneq ($(OS), OpenBSD)
